packages/devtools_app/lib/src/shared/console/widgets/help_dialog.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import '../../../shared/analytics/constants.dart' as gac;
+import '../../common_widgets.dart';
+
+class ConsoleHelpLink extends StatelessWidget {
+  const ConsoleHelpLink({Key? key}) : super(key: key);
+
+  static const _documentationTopic = gac.consoleHelp;
+
+  HelpButtonWithDialog _createButton() => HelpButtonWithDialog(
+        asAction: true,
+        gaScreen: gac.memory,
+        gaSelection: gac.topicDocumentationButton(_documentationTopic),
+        dialogTitle: 'Console Help',
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.end,
+          children: [
+            const Text(
+              r'''
+Use debug console to:
+
+1. Watch the standard output (stdout) of the application.
+2. Evaluate expressions for a paused or running application.
+3. Analyze inbound and outbound references for objects, dropped from memory heap snapshots.
+
+Assign previously evaluated objects to variable
+using $0, $1 … $5.
+Example: var x = $0
+''',
+            ),
+            MoreInfoLink(
+              // TODO(polina-c): create content at link.
+              url:
+                  'https://docs.flutter.dev/development/tools/devtools/console',
+              gaScreenName: '',
+              gaSelectedItemDescription:
+                  gac.topicDocumentationLink(_documentationTopic),
+            ),
+          ],
+        ),
+      );
+
+  void openDialog(BuildContext context) => _createButton().openDialog(context);
+
+  @override
+  Widget build(BuildContext context) => _createButton();
```

Comment:
this pattern is a bit strange. We shouldn't be creating a widget outside of the context of the widget tree (i.e. in `build`).

I think what we want is this:

Create a ConsoleHelpDialog stateless widget class and then instead of using `HelpButtonWithDialog`, use `HelpButton` directly:
```
  HelpButton(
      onPressed: () {
        ga.select(gaScreen, gaSelection);
        unawaited(
          showDialog(
            context: context,
            builder: (context) => ConsoleHelpDialog(),
          ),
        );
      },
      gaScreen: gaScreen,
      gaSelection: gaSelection,
    );
```

then above in evaluate.dart, instead of writing `ConsoleHelpLink().openDialog()` (which is another case where we are creating a widget outside of the context of the widget tree which we don't want), open the dialog directly:
```dart
       unawaited(
          showDialog(
            context: context,
            builder: (context) => ConsoleHelpDialog(),
          ),
        );
```